<ui version="4.0" >
 <class>Pref::Ui::GeneralWidget</class>
 <widget class="QWidget" name="Pref::Ui::GeneralWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>373</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabs" >
     <property name="currentIndex" >
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>365</width>
        <height>243</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>일반</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>재생목록</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <item>
           <widget class="QLabel" name="label_6" >
            <property name="text" >
             <string>파일을 열 때 자동으로 다음 파일들을 재생목록에 추가합니다.</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <property name="spacing" >
             <number>0</number>
            </property>
            <item>
             <widget class="QComboBox" name="auto_add_combo" >
              <property name="sizeAdjustPolicy" >
               <enum>QComboBox::AdjustToContents</enum>
              </property>
              <item>
               <property name="text" >
                <string>비슷한 이름의 파일</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>같은 경로의 모든 파일</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>다른 파일을 추가하지 않음</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5" >
         <property name="title" >
          <string>재생</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" >
          <item>
           <widget class="QCheckBox" name="pause_check" >
            <property name="text" >
             <string>최소화시에 일시 정지하기</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType" >
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>15</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="play_restored_check" >
              <property name="enabled" >
               <bool>false</bool>
              </property>
              <property name="text" >
               <string>복귀시에 자동으로 재생 시작하기</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line_2" >
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="restart_stopped_files_check" >
            <property name="text" >
             <string>재생중에 중지된 파일을 다시 열때 중지된 곳부터 재생하기</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2" >
      <property name="geometry" >
       <rect>
        <x>0</x>
        <y>0</y>
        <width>365</width>
        <height>243</height>
       </rect>
      </property>
      <attribute name="title" >
       <string>오디오/비디오</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>오디오</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="_2" >
            <item>
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>오디오 출력 드라이버</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="ao_combo" >
              <property name="editable" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" >
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="enabled" >
               <bool>true</bool>
              </property>
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Minimum" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text" >
               <string>볼륨 증폭률</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="amplification_spin" >
              <property name="accelerated" >
               <bool>true</bool>
              </property>
              <property name="suffix" >
               <string> %</string>
              </property>
              <property name="maximum" >
               <number>200</number>
              </property>
              <property name="value" >
               <number>100</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer" >
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="_4" >
            <item>
             <widget class="QCheckBox" name="reset_volume_check" >
              <property name="text" >
               <string>재생 시작할 때 소리를 다시 설정하기</string>
              </property>
              <property name="checked" >
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <layout class="QHBoxLayout" name="_5" >
              <item>
               <widget class="QLabel" name="label_7" >
                <property name="text" >
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSlider" name="volume_slider" >
                <property name="maximum" >
                 <number>100</number>
                </property>
                <property name="orientation" >
                 <enum>Qt::Horizontal</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2" >
         <property name="title" >
          <string>비디오</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <layout class="QHBoxLayout" >
            <item>
             <widget class="QLabel" name="label_2" >
              <property name="text" >
               <string>비디오 출력 드라이버</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="vo_combo" >
              <property name="editable" >
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer>
              <property name="orientation" >
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0" >
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
         <zorder></zorder>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pause_check</sender>
   <signal>toggled(bool)</signal>
   <receiver>play_restored_check</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>124</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel" >
     <x>134</x>
     <y>187</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
